stage1/init/pod.go

```diff
@@ -346,7 +346,7 @@ func (p *Pod) writeEnvFile(env types.Environment, id types.Hash) error {
 // all the constituent apps of the Pod
 func (p *Pod) PodToSystemd(interactive bool) error {
 	for _, am := range p.Apps {
-		ra := p.Manifest.Apps.Get(am.Name)
+		ra := common.GetAppByImageName(p.Manifest.Apps, am.Name)
```

Comment:
I think we can just iterate over p.Manifest.Apps now? As we can guarantee p.Manifest.Apps[i].App is non-nil after #1002